src/test/scala/com/github/music/of/the/ainur/almaren/Test.scala

```diff
@@ -62,9 +62,9 @@ class Test extends FunSuite with BeforeAndAfter {
 
   test(testSourceTargetJdbc(moviesDf), moviesDf, "SourceTargetJdbcTest")
   test(testSourceTargetJdbcUserPassword(moviesDf), moviesDf, "SourceTargetJdbcTestUserPassword")
-  test(testSourceFile("parquet","src/test/resources/sample_data/emp.parquet"),
+  test(testSourceTargetFile("parquet","src/test/resources/sample_data/emp.parquet",SaveMode.Overwrite,"src/test/resources/output_data/output.parquet"),
```

Comment:
keep test source FIle as separate test case and write another test case for the target file